client/web/src/enterprise/campaigns/list/CampaignsListEmpty.tsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react'
+
+export interface CampaignsListEmptyProps {
+    // Nothing for now.
+}
+
+export const CampaignsListEmpty: React.FunctionComponent<CampaignsListEmptyProps> = () => (
+    <div className="web-content">
+        <h2 className="my-5">Get started with campaigns</h2>
+        <h3 className="mb-3">Tutorials to help with your first campaign</h3>
+        <div className="row">
+            <div className="col-12 col-md-6 mb-2">
+                <div className="card h-100 p-2">
+                    <div className="card-body d-flex">
+                        <FindReplaceIcon className="mr-3" />
+                        <div>
+                            <h4>
+                                <a
+                                    href="https://docs.sourcegraph.com/campaigns/tutorials/search_and_replace_specific_terms"
+                                    rel="noopener"
+                                >
+                                    Finding and replacing exclusionary terms
+                                </a>
+                            </h4>
+                            <p className="text-muted mb-0">
+                                A Sourcegraph query plus a simple sed command creates changesets required to manage a
+                                large scale refactoring.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className="col-12 col-md-6 mb-2">
+                <div className="card h-100 p-2">
+                    <div className="card-body d-flex">
+                        <RefactorCombyIcon className="mr-3" />
+                        <div>
+                            <h4>
+                                <a
+                                    href="https://docs.sourcegraph.com/campaigns/tutorials/updating_go_import_statements"
+                                    rel="noopener"
+                                >
+                                    Refactoring with language aware search
+                                </a>
+                            </h4>
+                            <p className="text-muted mb-0">
+                                Using comby language aware search to refactor Go statements to a semantically
+                                equivalent, but clearer execution.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <p>
+            <a href="https://docs.sourcegraph.com/campaigns/tutorials" rel="noopener">
+                More tutorials
+            </a>
+        </p>
+        <div className="row mb-4">
+            <div className="col-12 col-md-4 mb-3">
+                <p>
+                    <strong>Quickstart</strong>
+                </p>
+                <p>Create your first Sourcegraph campaign in 10 minutes or less by altering a readme.</p>
```

Comment:
You don't create a campaign _by_ altering a README, but you can create a campaign _to_ alter READMEs.

I suggest leaving this off:

```suggestion
                <p>Create your first Sourcegraph campaign in 10 minutes or less.</p>
```